ui/src/external/monaco.flux.ts

```diff
@@ -0,0 +1,39 @@
+import initialize from 'src/external/monaco'
+
+initialize().then((monaco: any) => {
+  monaco.languages.register({id: 'Flux'})
+
+  monaco.languages.setMonarchTokensProvider('Flux', {
+    keywords: ['from', 'range', 'filter', 'to'],
+    tokenizer: {
+      root: [
+        [
+          /[a-z_$][\w$]*/,
+          {cases: {'@keywords': 'keyword', '@default': 'identifier'}},
+        ],
+      ],
+    },
+  })
+
+  monaco.editor.defineTheme('FluxTheme', {
+    base: 'vs',
+    inherit: false,
+    rules: [{token: 'keyword', foreground: 'ff00ff', fontStyle: 'bold'}],
+  })
+
+  monaco.languages.registerCompletionItemProvider('Flux', {
```

Comment:
🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥[ might wanna pull this out into it's own file too, but until then... ] 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥